src/main/java/duke/logic/Duke.java

```diff
@@ -18,6 +18,8 @@ public Duke(Ui ui) {
         this.ui = ui;
         try {
             tasks = STORAGE.deserialize();
+            tasks.updateRecurringTasks();
+            System.out.println("hello");
```

Comment:
This can be removed

Reply:
Okay will remove System.out.println("hello"); thank you

"tasks.updateRecurringTasks();" is to refresh the date of a task to a new date if the previous date has expired.